code/modules/mob/mob.dm

```diff
@@ -1593,3 +1593,24 @@ GLOBAL_LIST_INIT(holy_areas, typecacheof(list(
 	else
 		. = invoked_callback.Invoke()
 	usr = temp
+
+/mob/verb/give_kudos(mob/living/target as mob in oview())
+	set category = null
+	set name = "Give Kudos (OOC)"
+
+
+	if(target == src)
+		to_chat(src, "<span class='warning'>You cannot give kudos to yourself!</span>")
+		return
+
+	to_chat(src, "<span class='notice'>You've given kudos to [target]!</span>")
+
+	// Pretend we've always succeeded when we might not have.
+	// This should prevent people from using it to suss anything out about mobs' states
+	if(!mind || !target.mind)
+		return
+
+	var/our_uid = mind.UID()
```

Comment:
Just my thoughts, but this would probably be better ckey based

Reply:
Nah, I think you're right, since mind-code can be a little funky.